apps/backoffice/src/components/services/subscriptions-migration/migration-manager.tsx

```diff
@@ -0,0 +1,205 @@
+  data: MigrationData;
+};
+
+/** Renders subscriptions migrator helper _(from old Deleloper Portal)_ */
+export const MigrationManager = () => {
+  const { t } = useTranslation();
+  const [isImportModalOpen, setIsImportModalOpen] = useState(false);
+  const [importInProgress, setImportInProgress] = useState(false);
+  const [migrationStatusList, setMigrationStatusList] = useState<
+    Array<DelegateStatusPair>
+  >();
+
+  const {
+    data: migrationItemsData,
+    loading: migrationItemsLoading,
+    fetchData: migrationItemsFetchData
+  } = useFetch<MigrationItemList>();
+  const {
+    data: migrationDelegatesData,
+    loading: migrationDelegatesLoading,
+    fetchData: migrationDelegatesFetchData
+  } = useFetch<MigrationDelegateList>();
+
+  const handleOpenImportModal = () => {
+    setIsImportModalOpen(true);
+  };
+
+  const getMigrationOwnershipClaimsUrl = (delegateId: string) =>
+    `${
+      getConfiguration().API_BACKEND_BASE_URL
+    }/api/services/migrations/ownership-claims/${delegateId}`;
+
+  // direct axios fetch
+  const getDelegateMigrationStatus = async (delegateId: string) => {
+    try {
+      const { data, status } = await axios.get<MigrationData>(
+        getMigrationOwnershipClaimsUrl(delegateId)
+      );
+      return { data, status };
+    } catch (error) {
+      if (axios.isAxiosError(error)) {
+        return { data: error.message, status: -1 };
+      } else {
+        return { data: JSON.stringify(error), status: -1 };
+      }
+    }
+  };
+
+  // direct axios fetch
+  const postDelegateMigrationOwnershipClaims = async (delegateId: string) => {
+    try {
+      const { status } = await axios.post<void>(
+        getMigrationOwnershipClaimsUrl(delegateId)
+      );
+      return { status };
+    } catch (error) {
+      return { status: -1 };
+    }
+  };
+
+  const fetchDelegateStatusPairs = () => {
+    const pairs: Array<DelegateStatusPair> = [];
+    migrationDelegatesData?.delegates?.forEach(async delegate => {
+      const result = await getDelegateMigrationStatus(delegate.sourceId);
+      if (result.status === 200) {
+        pairs.push({
+          delegate,
+          data: result.data
+        });
+      }
+      setMigrationStatusList([...pairs]);
+    });
+  };
+
+  // claim owneship for selected delegates
+  const claimOwnership = async (delegates: string[]) => {
+    try {
+      setImportInProgress(true);
+      await Promise.all(delegates.map(postDelegateMigrationOwnershipClaims));
+      enqueueSnackbar(
+        buildSnackbarItem({
+          severity: "success",
+          title: t("notifications.success"),
+          message: ""
+        })
+      );
+    } catch (error) {
+      enqueueSnackbar(
+        buildSnackbarItem({
+          severity: "error",
+          title: t("notifications.exceptionError"),
+          message: JSON.stringify(error)
+        })
+      );
+    } finally {
+      setImportInProgress(false);
+      fetchMigrationLatestData();
```

Comment:
`fetchMigrationLatestData()` runs even when the above code fails, is it right?

Reply:
Yes right, it's a porting of old Developer Portal Frontend logic